### DISCUSSION TITLE
My Read and Write Permission button are disabled

### DISCUSSION THREAD

**Discussion by Durgesh4993** [community/community#125752] [Actions] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/98798977/274501f3-ea5b-461e-b9af-92a6298745ea)

**Comment by karnatigagan** [comment, marked as answer]
Hello Durgesh4993,

From your screenshot, it appears that the “Read and write permissions” option is disabled due to restrictions set at a higher level of administration (organization or main repository settings). Here are a few steps to resolve this issue:

	1.	Check Organization Policies: If your repository is part of an organization, there might be overarching security policies that restrict workflow permissions. You can check this by going to the organization’s settings, selecting “Actions” from the sidebar, and reviewing the “Permissions & security” settings.
	2.	Repository Settings: If you are the repository owner or have admin rights, you can check the repository settings. Navigate to “Settings,” then “Actions,” and under “General,” look for “Workflow permissions.” If changes are restricted here, you may need to adjust the settings or consult with other administrators.
	3.	Contact Admin: If you do not have the necessary permissions to change these settings, you may need to contact the repository or organization admin to request changes or get more information about the permission policies in place.
	4.	Review the Documentation: GitHub has detailed documentation on managing permissions for GitHub Actions, which might provide more insight or alternative solutions based on your specific setup.

It’s important to remember that these restrictions are usually in place to enforce security best practices, especially in larger projects or organizations. Always consider the security implications before changing access permissions.

I hope this helps!

**Comment by cNoveron** [comment]
Why was this answer marked as off topic????
